### DISCUSSION TITLE
Recommendations to follow bot accounts aren't helpful

### DISCUSSION THREAD

**Discussion by nschonni** [community/community#13298] [Other Feature Feedback, Questions, & Ideas]
Had a few recommendations for various names like
![image](https://user-images.githubusercontent.com/1297909/159612765-a6f3f9e0-39c0-4a4d-9f6f-b918abd25405.png)

May be hard to filter without a native way of marking bot accounts though

**Comment by sntwandy** [comment]
We need to implement a filter algorithm, to be able to filter those accounts that have been reported several times as "bots" and thus evaluate it so as not to show them anymore so we are cleaning up GitHub a bit. XD

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Comment by vectornguyen76** [comment]
Hello
